### DISCUSSION TITLE
how to zoom out polygon using CGAL

### DISCUSSION THREAD

**Discussion by qiangsun89** [CGAL/cgal#6699] [Q&A] [answered]
How to zoom out a polygon using CGAL?
![zoom_out](https://user-images.githubusercontent.com/46945831/175197968-5319c8bf-09ce-45e1-9a93-a97767e36d96.png)

**Comment by lrineau** [comment, marked as answer]
The operation you are looking for is an inward offset. Have a look at the chapter [2D Straight Skeleton and Polygon Offsetting](https://doc.cgal.org/latest/Straight_skeleton_2/index.html) of CGAL. It has examples about polygon offsetting.
